scripts/notify-docs/published-headless.js

```diff
@@ -0,0 +1,30 @@
+const {Octokit} = require('@octokit/rest');
```

Comment:
I love `octokit` ;) 

Makes it super easy to do any operation against github repos.